pkgs/applications/science/biology/dcm2niix/default.nix

```diff
@@ -17,7 +24,23 @@ stdenv.mkDerivation rec {
   };
 
   nativeBuildInputs = [ cmake git ];
-  buildInputs = [ libyamlcpp ];
+  buildInputs =
+    lib.optional batchVersion [ libyamlcpp ]
```

Comment:
This library is only used if you enable batch processing support:

https://github.com/rordenlab/dcm2niix/blob/003f0d19f1e57b0129c9dcf3e653f51ca3559028/SuperBuild/SuperBuild.cmake#L88-L107.